addons/mass_mailing_sms/tests/test_mailing_unsubscribe.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests.common import users
+from odoo.addons.mass_mailing_sms.tests.common import MassSMSCommon
+import werkzeug.urls
+import random
+from odoo.tests import HttpCase
+
+class TestMailingListSms(HttpCase,MassSMSCommon):
+
+    @users('user_marketing')
+    def test_mailing_list_action_send_sms(self):
+
+        partner = self.env['res.partner'].create({
+            'name': 'Test Partner',
+            'phone': '+91 1234657890',
+        })
+
+        mailing = self.env['mailing.mailing'].create({
+            'name': 'TestMailing',
+            'subject': 'Test',
+            'mailing_type': 'sms',
+            'body_plaintext': 'Coucou hibou',
+            'mailing_model_id': self.env['ir.model']._get('res.partner').id,
```

Comment:
Would also add a domain (like id, =, partner.id) so that we avoid creating traces for all current partners in db